tests/marker_test.go

```diff
@@ -60,4 +67,51 @@ var _ = Describe("bridge-marker", func() {
 			}, 20*time.Second, 5*time.Second).Should(Equal(false))
 		})
 	})
+
+	Describe("pod requiring bridge resource", func() {
+		It("should be started only when bridge is available on node", func() {
+
+			resourceName := tests.GenerateResourceName()
+			requiredResourceCount := "1"
+			requiredResources := corev1.ResourceList{
+				resourceName: resource.MustParse(requiredResourceCount),
+			}
+
+			podReq := tests.PodSpec(TestPodName, requiredResources)
+
+			_, err := clientset.CoreV1().Pods(DefaultNamespace).Create(podReq)
+			Expect(err).ToNot(HaveOccurred())
+
+			Eventually(func() bool {
```

Comment:
can you please create a function for this function it repeat itself 3 times.

you can send the podName and the phase

Reply:
Note that we are making a different check each time.
I created a function (it should probably go into utils, I'll move it when removing the debug code), but to me it does not look much better than the original code.